homeassistant/components/webostv/__init__.py

```diff
@@ -138,15 +139,30 @@ async def async_on_stop(event):
         client.clear_state_update_callbacks()
         await client.disconnect()
 
-    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, async_on_stop)
+    async def async_load_platforms(_):
+        """Load platforms and event listener."""
+        await async_connect(client)
 
-    await async_connect(client)
-    hass.async_create_task(
-        hass.helpers.discovery.async_load_platform("media_player", DOMAIN, conf, config)
-    )
-    hass.async_create_task(
-        hass.helpers.discovery.async_load_platform("notify", DOMAIN, conf, config)
-    )
+        if client.connection is None:
+            async_call_later(hass, 60, async_load_platforms)
```

Comment:
It can be nice to use a back off mechanism like this:
https://github.com/home-assistant/core/blob/15569f1e7fc2fa79de88f8c933973e1bd818f517/homeassistant/components/tibber/__init__.py#L51

Reply:
I personally would be against a back-off mechanism with a max of 15 minutes because I use it to monitor how much TV my kid has watched. Having an offset of up to 15 minutes is not much if they're allowed 30 minutes in total ;)